tests/datadriven/test_rna_quantification.py

```diff
@@ -0,0 +1,149 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import ga4gh.datamodel as datamodel
+import ga4gh.datamodel.datasets as datasets
+import ga4gh.protocol as protocol
+import ga4gh.datamodel.rna_quantification as rna_quantification
+import tests.datadriven as datadriven
+import ga4gh.exceptions as exceptions
+
+
+_rnaQuantTestData = {
+    "id": "ZHM6RU5DRkYzMDVMWkI",
```

Comment:
I've moved away from quoting actual IDs in these tests: Since the ID generation mechanism is still very much in flux, these _will_ change. Instead, consider similar code from `datadriven/test_sequenceAnnotations.py`:
```
_gencodeV21Set1TestData = {
    "featureSetName": "gencodeV21Set1",
    "referenceName": "chr1",
    "totalFeatures": 543,
    "sampleFeatureId": 4397111632,
    "sampleParentId": 4397111312,
    "sampleStart": 804776,
    "sampleEnd": 804832,
    "sampleStrand": protocol.Strand.POS_STRAND,
    "sampleSiblings": 5,
    "region": [0, 2**32],
    "ontologyRestriction": ["gene", ],
    "featuresWithOntology": 58
}
```
Note that the Ids there are DATABASE IDs, not compoundIds. I generate the compoundIDs on the fly in the test code as follows:
```
        idString = _getFeatureCompoundId(
            _datasetName,
            self._testData["featureSetName"],
            self._testData["sampleFeatureId"])
```